tests/integration/store/conftest.py

```diff
@@ -0,0 +1,84 @@
+import sys
+import json
+import pytest
+import uuid
+import dash
+from dash.dependencies import Input, Output, State
+from dash.exceptions import PreventUpdate
+import dash_core_components as dcc
+import dash_html_components as html
+
+UUID = "store-test-{}".format(uuid.uuid4().hex)
+
+
+@pytest.fixture(scope="module")
+def store_app():
```

Comment:
it's another way to define the app, not inside tests but provide the app as another fixture. 

pros: share the same app among tests, make the test terser, even without import.
cons: you might need to open two windows to have a better understanding of the test.